public/waivers/live/ApGuestWaiver.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
@evan-mccarty Any reason these need to be whole html files iframe'd in?  Seems like they should just be a react component that takes the waiver text as prop, especially since there is so much shared code between these two files.  Also I discussed with Andrew and I think we will at some point have the waiver text come from the backend, although that's not a stopship

Reply:
I did this for two reasons. First, it's easier printing the waiver text from the html in the iframe than making a separate print view in TS. Second is a rudimentary version control until we implement a permanent solution.

All the file has is the SVG logo and the waiver text so its pretty small.

If you can think of a more elegant solution,  I'm all ears. (and /or keyboard)

EM

Reply:
Oh neat, I didn't see the print button.  Yeah that sounds fine then, probably simple than trying to figure out how to just print a single div